internal/stage_handshake.go

```diff
@@ -77,8 +76,6 @@ func testHandshake(stageHarness *test_case_harness.TestCaseHarness) error {
 			logger: logger,
 	})
 
-	time.Sleep(1 * time.Millisecond) // for deterministic output ordering in fixtures
```

Comment:
no longer needed, resolved this by setting `debug = false` in `with_peer_check` test